getting_started/step_by_step/your_first_game.rst

```diff
@@ -22,23 +22,29 @@ final result:
 .. image:: img/dodge_preview.gif
 
 **Why 2D?** 3D games are much more complex than 2D ones. You should stick to 2D
-until you have a good understanding of the game development process.
+until you have a good understanding of the game development process and how to
+use Godot.
 
 Project setup
 -------------
 
 Launch Godot and create a new project. Then, download
-:download:`dodge_assets.zip <files/dodge_assets.zip>` - the images and sounds you'll be
-using to make the game. Unzip these files to your project folder.
+:download:`dodge_assets.zip <files/dodge_assets.zip>`. This contains the
+images and sounds you'll be using to make the game. Unzip these files in your
+project folder.
 
 .. note:: For this tutorial, we will assume you are familiar with the
-          editor. If you haven't read :ref:`doc_scenes_and_nodes`, do so now
+          Godot editor. If you haven't read :ref:`doc_scenes_and_nodes`, do so now
           for an explanation of setting up a project and using the editor.
 
-This game will use portrait mode, so we need to adjust the size of the
+This game is designed for portrait mode, so we need to adjust the size of the
 game window. Click on Project -> Project Settings -> Display -> Window and
 set "Width" to ``480`` and "Height" to ``720``.
 
+Also in this section, under the "Stretch" options, set ``Mode`` to “2d” and
+``Aspect`` to “keep”. This ensures that the game scales consistently on
+different sized screens.
```

Comment:
```suggestion
Also in this section, under the "Stretch" options, set ``Mode`` to "2d" and
``Aspect`` to "keep". This ensures that the game scales consistently on
different sized screens.
```

I suggest changing the double quotes to ASCII ones for consistency with the rest of the tutorial.

Reply:
Thanks - not sure how those snuck in...